test/counts.jl

```diff
@@ -114,3 +114,19 @@ pm = proportionmap(x, weights(w))
 @test pm["a"] ≈ (5.5 / 13.5)
 @test pm["b"] ≈ (4.5 / 13.5)
 @test pm["c"] ≈ (3.5 / 13.5)
+
+# testing small bits type
+bx = [true, false, true, true, false]
+@test countmap(bx) == Dict(true => 3, false => 2)
+
+tx = UInt8[0, 10, 255, 34, 10]
+@test countmap(tx) == Dict(0 => 1, 10 => 2, 255 => 1, 34 => 1)
+
+tx = UInt16[0, 8, 64000, 27, 8]
```

Comment:
Why `64000`? It would make more sense to test `65535`, to ensure no overflow happens. Same for other types (better write this as `typemax(T)`, and also include `typemin(T)` while you're at it). As I suggested before, put these inside a `for T in (UInt8, ...` loop (using the same values for all except for typemin/typemax).